src/main/java/org/pawpal/dto/PetDTO.java

```diff
@@ -0,0 +1,57 @@
+package org.pawpal.dto;
+
+public class PetDTO {
+    private Long id;
+    private String name;
+    private String breed;
+    private int age;
+    private String email;
+
+    public PetDTO(Long id, String name, String breed, int age, String ownerId) {
```

Comment:
adauga un coment pentru treaba this.email = ownerId;
emailul ii defapt id sau id ul ii defapt email?